setup.py

```diff
@@ -77,15 +77,19 @@ def run(self): # noqa
                             'firmware_policy_tool=edk2toolext.windows.policy.firmware_policy_tool:main',
                             'edk2_capsule_tool=edk2toolext.capsule.capsule_tool:main',
                             'versioninfo_tool=edk2toolext.versioninfo.versioninfo_tool:main',
-                            'validate_image_tool=edk2toolext.image_validation:main']
+                            'validate_image_tool=edk2toolext.image_validation:main',
+                            'secureboot_audit=edk2toolext.windows.secureboot.secureboot_audit:main']
```

Comment:
nit: add a comma at the end of this list. This is so that when we add a new tool, it shows we are only editing one line instead of two like what we see above (because I failed to add a comma to validate_image_tool when I added that 🥇 )